scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -97,6 +109,12 @@
         "dcrName":"[if(greater(length(variables('dcrNameFull')), 64), substring(variables('dcrNameFull'), 0, 64), variables('dcrNameFull'))]",
         "associationName":  "ContainerInsightsExtension",
         "dataCollectionRuleId": "[resourceId(variables('clusterSubscriptionId'), variables('clusterResourceGroup'), 'Microsoft.Insights/dataCollectionRules', variables('dcrName'))]",
+        "dceName": "[variables('dcrName')]",
```

Comment:
do we want to concat some string to indicate this is the DCE?

Reply:
> do we want to concat some string to indicate this is the DCE as dcr, dcea,ampls depends on it?
> adding additional string will cause the name going over 64 chars.  type can identified by the resource type even though name of the resource is same.